src/components/Style/question.css

```diff
@@ -10,13 +10,32 @@
 }
 .currentQuestionContainer {
   display: grid;
-  grid-template-columns: 75% 25%;
+  justify-content: space-between;
+  align-items: flex-end;
+  grid-template-columns: 65% 35%;
   grid-gap: 0;
   margin-bottom: 1rem;
 }
 .productQuestion {
-  font-size: 0.75rem;
+  font-size: 0.85rem;
 }
 .productValue {
   font-size: 1rem;
 }
+.imgContainer {
+  display: grid;
```

Comment:
There is a bug on Chrome, the image is 12px high. Removing the grid property seems to fix the issue.